### ISSUE TITLE
TELLER_APP_ID

### ISSUE THREAD

**Issue by Logdawg1606** [songyuew/teller#2]
Your readme says to change the "TELLER_APP_ID" to match your personal teller app id in the code, however that "TELLER_APP_ID" does not appear anywhere in the code in order to replace.

**Comment by songyuew** [comment]
Please go to [public/index.js](https://github.com/songyuew/teller/blob/main/public/index.js) to modify `APPLICATION_ID` field. Sorry for the confusion.

**Comment by Logdawg1606** [comment]
thanks so much i was able to get this working.

**Comment by omnicortex** [comment]
We should update the README to clarify this.

**Comment by songyuew** [comment]
> We should update the README to clarify this.

@omnicortex Yes, thank you very much for the suggestion. Will do accordingly!
